content/community/events/2026-01-26-hpcasia-tutorial.md

```diff
@@ -0,0 +1,29 @@
+# HPC Asia / SCA 2026 Kokkos Tutorial
+
+{{< image src="img/events/2026/HPCAsia-Osaka.jpg" style="float: center; height=10" caption="Osaka castle. By 663highland - Own work, CC BY 2.5">}}
```

Comment:
How come we are not using a header like
```
---
authors: ["kokkos-team"]
title: "Field report for the Kokkos User Group Meeting 2025"
date: 2025-05-09
tags: ["blog"]
thumbnail: img/blog/2025/KUG2025-1.jpg
---

Reply:
we can add that. But I thought you wanted to revamp the whole page anyway?